### ISSUE TITLE
[Question] mounting UIControl action outside of it's container

### ISSUE THREAD

**Issue by hermanccw** [schibsted/layout#58]
I have the following layout
```
<UITableViewHeaderFooterView
      backgroundView.backgroundColor="#fff"
      height="50"
      reuseIdentifier="tableFooter">
      
      <UIButton
        width="100%"
        top="10"
        title="Continue"
        titleColor="black"
        touchUpInside="continuePressed"/>
    </UITableViewHeaderFooterView>
```

I do have a `continuePressed` function in the viewController, but I am seeing the following error 
```
Layout error: UITableViewHeaderFooterView does not respond to continuePressed.

If the method exists, it must be prefixed with @objc or @IBAction to be used with Layout in UIButton

```

Is there a way that the action can be mounted to the controller instead of the parent view?

**Comment by nicklockwood** [comment]
This is an edge case I didn't consider. Since the header isn't part of the Layout hierarchy (it's created and managed by the UITableView internally) it doesn't have access to the view controller.

I think this is a reasonable use-case though. I'll see if I can find a solution.

In the meantime, you'll have to bind the action manually in your `tableView(_:viewForHeaderInSection:)` method. There's no particularly elegant way to grab a reference to the UIButton though. The best option is probably to use `viewWithTag(_:)`, or to subclass `UITableViewHeaderFooterView` so you can give it an outlet for the button.

**Comment by nicklockwood** [comment]
@hermanccw fixed in Layout 0.5.3
